src/diffusion/include/antioch/molecular_binary_diffusion.h

```diff
@@ -122,10 +113,10 @@ namespace Antioch{
 
           //! \return molecular binary diffusion coefficient
           template <typename StateType>
-          const 
-          ANTIOCH_AUTO(StateType) 
-                binary_diffusion(const StateType & T, const StateType & cTot) const
-          ANTIOCH_AUTOFUNC(StateType,_coefficient * _interp.interpolated_value(T) / cTot )
+          const
+          ANTIOCH_AUTO(StateType)
+                binary_diffusion(const StateType & T, const StateType & molar_density) const
+          ANTIOCH_AUTOFUNC(StateType,_coefficient * _interp.interpolated_value(T) / molar_density )
 
            //!
```

Comment:
THanks. The only thing I did in the species classes was make them adhere to an interface and didn't try to clean up. We'll do that at a later time... hopefully. >_>